package.json

```diff
@@ -118,7 +118,7 @@
     "gaba": "^1.9.0",
     "human-standard-token-abi": "^2.0.0",
     "jazzicon": "^1.2.0",
-    "json-rpc-engine": "^5.1.5",
+    "json-rpc-engine": "rekmarks/json-rpc-engine#ordered-batch",
```

Comment:
This also needs to be merged into the main repo and published

Reply:
Yeah, waiting for this to be merged: MetaMask/json-rpc-engine#21